src/app/public/modules/demo/demo.component.ts

```diff
@@ -20,12 +25,23 @@ import {
 })
 export class SkyDocsDemoComponent {
 
+  @Input()
+  public set alignContents(value: SkyDocsDemoContentAlignment) {
```

Comment:
This will allow consumers to specify the alignment of the demo, either "left" or "center" (default is "left").